Makefile

```diff
@@ -32,20 +32,16 @@ build:
 # initial impl. Feel free to override. Please keep ARTIFACT_ROOT coming from env though. CI/CD pipeline relies on this
 build-all:
 	echo "GOPATH=$(GOPATH)"
-	mkdir -p ${ARTIFACT_ROOT}
-	mkdir -p ${ARTIFACT_ROOT_AMD64}
-	mkdir -p ${ARTIFACT_ROOT_ARM64}
-	mkdir -p ${ARTIFACT_ROOT_MAC}
-	mkdir -p ${ARTIFACT_ROOT_WIN}
-	GOOS=linux GOARCH=amd64 $(GO) build -o ${ARTIFACT_ROOT_AMD64}/$(BINARY)-linux-amd64 $(GOFLAGS); cd ${ARTIFACT_ROOT_AMD64}; ln -s $(BINARY)-linux-amd64  $(BINARY)
-	GOOS=linux GOARCH=arm64 $(GO) build -o ${ARTIFACT_ROOT_ARM64}/$(BINARY)-linux-arm64 $(GOFLAGS); cd ${ARTIFACT_ROOT_ARM64}; ln -s $(BINARY)-linux-arm64 $(BINARY)
-	GOOS=darwin GOARCH=amd64 $(GO) build -o ${ARTIFACT_ROOT_MAC}/$(BINARY)-mac $(GOFLAGS); cd ${ARTIFACT_ROOT_MAC}; ln -s $(BINARY)-mac $(BINARY)
-	GOOS=windows GOARCH=amd64 $(GO) build -o ${ARTIFACT_ROOT_WIN}/$(BINARY)-win.exe $(GOFLAGS); cd ${ARTIFACT_ROOT_WIN}; ln -s  $(BINARY)-win.exe $(BINARY).exe
 
-	tar -czvf ${ARTIFACT_ROOT}/$(BINARY)-linux-amd64-$(VERSION).tar.gz Attribution.txt LICENSE -C ${ARTIFACT_ROOT_AMD64} $(BINARY)-linux-amd64 $(BINARY)
-	tar -czvf ${ARTIFACT_ROOT}/$(BINARY)-linux-arm64-$(VERSION).tar.gz Attribution.txt LICENSE -C ${ARTIFACT_ROOT_ARM64} $(BINARY)-linux-arm64 $(BINARY)
-	tar -czvf ${ARTIFACT_ROOT}/$(BINARY)-mac-$(VERSION).tar.gz Attribution.txt LICENSE -C ${ARTIFACT_ROOT_MAC} $(BINARY)-mac $(BINARY)
-	zip -j --symlinks ${ARTIFACT_ROOT}/$(BINARY)-win-$(VERSION).zip ${ARTIFACT_ROOT_WIN}/$(BINARY)-win.exe ${ARTIFACT_ROOT_WIN}/$(BINARY).exe Attribution.txt LICENSE
+	GOOS=linux GOARCH=amd64 $(GO) build -o ${ARTIFACT_ROOT}/$(BINARY)-linux-amd64 $(GOFLAGS)
```

Comment:
Makefile needs further clean-up

1) would be good as part of the clean target to remove or at the very least rename (by adding a .bak extension} the $HOME/.edgex-cli/configuration.toml file. This tripped me up during tests when I accidentally undid a previous Makefile change of mine despite cleaning!

2) need to clean up the install target, basically deleting the copy of the res/configuration.toml file

3) the build target places the binary in the top level edgex-cli git repo versus the bin directory of the same. Would not your copy code that executes on first run then have difficulty finding the sample file .. the sort of issue you needed to tackle in your IDE environment?

Reply:
@mkbhanda  Thank you for  your code review 
I have missed a lot of things 
I think I have addressed all of them but only this
`the build target places the binary in the top level edgex-cli git repo versus the bin directory of the same. Would not your copy code that executes on first run then have difficulty finding the sample file .. the sort of issue you needed to tackle in your IDE environment?`
There is not problem with this because ./res/sampl-conifguration.toml can be found by the executable (actually this is the same case as with the IDE - already handled in the code)